lib/raven/client.rb

```diff
@@ -23,14 +22,6 @@ def initialize(configuration)
     end
 
     def send(event)
-      unless configuration.send_in_current_environment?
-        configuration.log_excluded_environment_message
-        return
-      end
-
-      # Set the project ID correctly
-      event.project = self.configuration.project_id
-
```

Comment:
I couldve sworn I left a comment on this!

* [setting the project ID is now the responsibility of the Event](https://github.com/getsentry/raven-ruby/pull/270/files#diff-8338fea26aee624929fed65e2d301325R44)
* Excluding due to the environment is already happening in Raven::Base, although after I saw how people use Raven::Client.send in background jobs, I think I may need to reconsider this.